src/wp-includes/block-patterns.php

```diff
@@ -158,11 +158,38 @@ function _register_core_block_patterns_and_categories() {
 	);
 }
 
+/**
+ * Normalize the pattern properties to camelCase.
+ *
+ * The API's format is snake_case, `register_block_pattern()` expects camelCase.
+ *
+ * @since 6.2.0
+ * @access private
+ *
+ * @param array $pattern Pattern as returned from the Pattern Directory API.
+ * @return array Normalized pattern.
+ */
+function _normalize_remote_pattern( $pattern ) {
```

Comment:
A search of the wp.org repo shows:

* [0 plugins](https://wpdirectory.net/search/01GR9VF5A5J7SFYT4CSPWNCNH9) with this same function name
* [0 themes](https://wpdirectory.net/search/01GR9VF5A5J7SFYT4CSPWNCNH9) with this same function name

The name follows the naming patterns in this file.

Note: There could be a risk for a naming conflict in the wild.

Reply:
Open to any suggestions :)

Reply:
Would `_normalize_remote_block_pattern()` help? The other functions in this file refer to "block" and/or "theme" in their names. (Also, there are [0 plugins](https://wpdirectory.net/search/01GRCP4M28CKZR6D2E6K0DMDDT) and [0 themes](https://wpdirectory.net/search/01GRCP5RJCZTXBMDC7JJAHAYEB) using this.)

Reply:
I renamed to `_normalize_remote_block_pattern`.

Reply:
See https://github.com/WordPress/wordpress-develop/pull/3967#pullrequestreview-1285837538, we should not be using underscore prefixes in Core. This is something that I was already advised by seasoned committers several years ago. While occasionally it gets overlooked, it's not a good practice. The recommendation is to simply mark the function with `@access private`.

I've updated this in https://github.com/WordPress/wordpress-develop/pull/3967/commits/6f39a6858df3f0f9f4ff4add23f3c0af10937afe.